guides/common/modules/proc_synchronizing-the-satellite-tools-repository.adoc

```diff
@@ -6,22 +6,50 @@ Use this section to synchronize the {project-client-name} repository from the Re
 This repository provides the `katello-agent`, `katello-host-tools`, and `puppet` packages for clients registered to {ProjectServer}.
 endif::[]
 
+ifndef::satellite[]
```

Comment:
This file is only used for Satellite (see the filename as well) so you can drop all non-Satellite bits in this file.

Reply:
Addressed :+1: